cluster/sbin/snappy-start-all.sh

```diff
@@ -34,6 +34,21 @@ if [ -f "${MEMBERS_FILE}" ]; then
   rm $MEMBERS_FILE
 fi
 
+SERVERS_STATUS_FILE="$SNAPPY_HOME/work/members-status.txt"
+if [ -f "${SERVERS_STATUS_FILE}" ]; then
+  rm $SERVERS_STATUS_FILE
+fi
+
+CAN_START_LEAD=0
+function checkIfOkayToStartLeads() {
+  while  read -r line || [[ -n "$line" ]]; do
+    read exitstatus member <<< $line
+    if [ "x$exitstatus" = "x0" -o "x$exitstatus" = "x10" ]; then
+      CAN_START_LEAD=1
+    fi
+  done < $SERVERS_STATUS_FILE
+}
+
```

Comment:
@kneeraj will be better to move this to snappy-lead.sh or snappy-leads.sh because k8s setup does not use start-all. Check with @dshirish as to which one it actually uses so will be better to move this code block and its caller to that script.